src/client/screens/Feed/index.js

```diff
@@ -67,10 +68,10 @@ class FeedPage extends React.Component {
           let posts = JSON.parse(xhr.response).post;
 
           posts.reverse(posts.timeStamp);
-          posts = posts.map(function(item, index){
-            let newDate = new Date(item.timeStamp)
+          posts = posts.map(function(item, index) {
+            let newDate = new Date(item.timeStamp);
 
-            item.timeStamp = newDate.toString();
+            item.timeStamp = formatDate(newDate);
```

Comment:
timestamp is a timestamp, but you create a datetime string, so in this case we should extend the original object or add a new property and delete the previous one. If in the future somebody want to use your component he/she will think this is a timestamp